go.mod

```diff
@@ -11,6 +11,11 @@ require (
 	golang.org/x/oauth2 v0.11.0
 )
 
+require (
```

Comment:
Can you run `go mod tidy`?  I think that might combine this with the block below.

Reply:
done!